cards/src/main/resources/cards/custom/group0/other_death_knight/classic/minion_tombstone_terror.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Tombstone Terror",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 4,
+  "baseHp": 7,
+  "rarity": "RARE",
+  "description": "Battlecry: Gain the text of a random enemy minion that died this game.",
+  "battlecry": {
+    "targetSelection": "NONE",
+    "spell": {
+      "class": "custom.CastUsingCardSourceSpell",
+      "cardSource": {
+        "class": "GraveyardActorsSource",
+        "targetPlayer": "OPPONENT"
+      },
+      "cardFilter": {
+        "class": "CardFilter",
+        "cardType": "MINION"
+      },
+      "randomTarget": true,
+      "spell": {
+        "class": "MetaSpell",
+        "spells": [
+          {
+            "class": "custom.AddActorEffectsToTargetActorSpell",
```

Comment:
Well at the moment it specifically doesn't support anything but MinionCards, so it would be some work to alter it to do what this spell already seems to do well enough